client/src/Invitation.js

```diff
@@ -0,0 +1,110 @@
+import React from "react";
+import { makeStyles } from "@material-ui/core/styles";
+import Button from "@material-ui/core/Button";
+import TextField from "@material-ui/core/TextField";
+import Dialog from "@material-ui/core/Dialog";
+import DialogActions from "@material-ui/core/DialogActions";
+import DialogContent from "@material-ui/core/DialogContent";
+import { CopyToClipboard } from "react-copy-to-clipboard";
+
+export default function FormDialog() {
+  const useStyles = makeStyles((theme) => ({
+    margin: {
+      margin: theme.spacing(3),
+    },
+    extendedIcon: {
+      marginRight: theme.spacing(1),
+    },
+  }));
+  const classes = useStyles();
+  const inputRef = React.createRef();
+  const [emails, setEmail] = React.useState("");
+  const [open, setOpen] = React.useState(false);
+  const [uniqueID, setID] = React.useState(
+    "https://www.EKLN-messenger.com/join/" +
+      (Math.floor(Math.random() * 90000000) + 10000000)
+  );
+  const [emailList, setEmailList] = React.useState([]);
+
+  /*generate unique id for URL */
+
+  /*Open dialog function */
+  const handleClickOpen = () => {
+    setOpen(true);
+  };
+
+  /*close dialog function*/
+  const handleClose = () => {
+    setOpen(false);
+  };
+
+  /*Set whatever in input to emails*/
+  const inputEmail = (e) => {
+    setEmail(e.target.value);
+  };
+
+  /*push emails to emailList*/
+  const addEmail = () => {
+    if (emailList.findIndex((email) => email == emails) < 0)
+      setEmailList([...emailList, emails]);
+  };
+
+  return (
+    <div>
+      <Button variant="outlined" color="primary" onClick={handleClickOpen}>
+        Invite Friends
+      </Button>
+      <Dialog
+        open={open}
+        onClose={handleClose}
+        aria-labelledby="form-dialog-title"
+        fullWidth={true}
+      >
+        <p className="invitation-dialog-title">
```

Comment:
We can use Material UI's Typography here and below instead of p: https://material-ui.com/components/typography/